### PR TITLE
Create login page and signup page without database

### PR DESCRIPTION
**Pull request by moshaosama**

### CODE REVIEW

File: Client/package-lock.json
Comment (thisAAY) [inline Client/package-lock.json:1]: I think this file can be added to gitignore right?

**Review by thisAAY — CHANGES_REQUESTED** [review]
I don't have knowlage with JS or web in general so I will try to review it as just code for me 

Next time please have a meaningful name for the branch

File: Client/src/Components/Account.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+const Account = ({ Title }: account) => {
+  return (
+    <>
+          <h1 className="font-bold text-3xl">TaskMo4a</h1>
+          <div className="my-10">
+            <p className="text-xl">{Title} your account</p>
+            <p>
+              Unlock exclusive perks - Sign up now for early access and special
+              offers!
```
Comment (thisAAY) [inline Client/src/Components/Account.tsx:12]: Can we have any kind of localization? What if I want to have all texts in Arabic and English? and it switches automatically based on region?

File: Client/package-lock.json
Comment (moshaosama) [reply Client/package-lock.json:1]: Okay,
when Clone this project in second time i will install him okay

File: Client/src/Components/Account.tsx (hunk 1 above)
Comment (moshaosama) [reply Client/src/Components/Account.tsx:12]: i can't understand this,
but i found this Text in Template that i am trying to copy, can you understand me?

File: Client/package-lock.json
Comment (thisAAY) [reply Client/package-lock.json:1]: I would prefere to have this done this PR
you can find .gitignore file for your projects from here
http://gitignore.io/

let me know what you might do with this

File: Client/src/Components/Account.tsx (hunk 1 above)
Comment (thisAAY) [reply Client/src/Components/Account.tsx:12]: Please research Web localization, and let's discuss it after

File: Client/package-lock.json
Comment (moshaosama) [reply Client/package-lock.json:1]: i know this file, worked on him before but i forget edit this file
when write this npm i react app in terminal he is install auto in structure Files

File: Client/src/Components/Account.tsx (hunk 1 above)
Comment (moshaosama) [reply Client/src/Components/Account.tsx:12]: Okay i will research about this topic

**Pull request merged**